src/Domain/Tags/Faces.hpp

```diff
@@ -0,0 +1,119 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Structure/DirectionMap.hpp"
+
+namespace domain {
+namespace Tags {
+
+/// The `Tag` on element faces
+template <size_t Dim, typename Tag>
+struct Faces : db::PrefixTag, db::SimpleTag {
+  using tag = Tag;
+  using type = DirectionMap<Dim, typename Tag::type>;
+};
+
+}  // namespace Tags
+
+namespace FacesTag_detail {
+template <typename T, typename = std::void_t<>>
+struct get_volume_tags_impl {
+  using type = tmpl::list<>;
+};
+template <typename T>
+struct get_volume_tags_impl<T, std::void_t<typename T::volume_tags>> {
+  using type = typename T::volume_tags;
+};
+}  // namespace FacesTag_detail
+
+/// Retrieve the `T::volume_tags`, or `tmpl::list<>` if `T` doesn't have the
+/// type alias
+template <typename T>
+using get_volume_tags =
```

Comment:
This doesn't have a name clearly relating to faces and is in a generic namespace (domain), so I worry about name collisions.  Maybe put it in the detail namespace or, alternatively, move it to a more general file.

Reply:
I just deleted this and removed the default :)